deploy/helm/sumologic/README.md

```diff
@@ -573,8 +573,8 @@ The following table lists the configurable parameters of the Sumo Logic chart an
-| `pvcCleaner.job.image.repository`                                                | Image repository for pvcCleaner docker containers.                                                                                                                                                                                                                                                                                                                   | `public.ecr.aws/sumologic/kubernetes-tools`                                                                                                                                          |
-| `pvcCleaner.job.image.tag`                                                       | Image tag for pvcCleaner docker containers.                                                                                                                                                                                                                                                                                                                          | `2.15.0`                                                                                                                                                                             |
+| `pvcCleaner.job.image.repository`                                                | Image repository for pvcCleaner docker containers.                                                                                                                                                                                                                                                                                                                   | `public.ecr.aws/sumologic/kubectl`                                                                                                                                                   |
+| `pvcCleaner.job.image.tag`                                                       | Image tag for pvcCleaner docker containers.                                                                                                                                                                                                                                                                                                                          | `1.26.3`                                                                                                                                                                             |
```

Comment:
Technically, that sounds like a breaking change (although I doubt that many customers are using it now), do we have any procedures for that?

Reply:
It is breaking change, as we change `mount` property of the pod. I don't think we have any procedure for that. We can add warning message for helm chart before `3.5.0` to ask customers to remove the current jobs

Reply:
Sounds good, but I guess that it still should be discussed once @perk-sumo is back.

Reply:
If someone changed the image tag, for example. 
I doubt someone is using this with overridden image tag or image name, but still.

Reply:
I double checked and pod spec in cronjob doesn't seem to be immutable (at least volume configuration is not)

Reply:
> If someone changed the image tag, for example.
> I doubt someone is using this with overridden image tag or image name, but still.

If you use a different tools image version, this will still work - the only requirements are bash and kubectl. If someone put something completely custom here, I don't think we support that.